src/main/java/com/solvd/laba/MedicalForms/AppointmentForm.java

```diff
@@ -0,0 +1,47 @@
+package com.solvd.laba.MedicalForms;
+
+import java.util.Arrays;
+
+import com.solvd.laba.Utils.Utils;
+
+public class AppointmentForm extends MedicalForms {
+	protected String sick; 
+	protected String reasonForVisit; 
+	protected String time; 
+	
+	public AppointmentForm(String age, String name, String time, String sick, String reasonForVisit) {

```

Comment:
patientAge, patientAge.
But it would be better to have a Patient as an object of a class, so in case one patient wants to make several appointments we will have one object that we can use.